### DISCUSSION TITLE
Misunderstanding on what's the output of the command "brew man"

### DISCUSSION THREAD

**Discussion by RogerioSoares96** [Homebrew/discussions#944] [Getting started]
Greetings everyone in git, i'm a newbie in computer terminal and just installed homebrew. By mistake I misplaced the word "man" after brew thinking that I would get the manpages. I ended up installing a bundler and other stuff I could not get a finger on. Could someone clarify me what I did?

**Comment by SMillerDev** [comment]
You made brew regenerate it's manpages. The command you're looking for is `man brew`

**Reply by RogerioSoares96** [reply]
thank you so much @SMillerDev I really appreciate the answer and the pointer !🙂 by curiosity, when you say regenerate the manpages, you are referring to a refresh in the included packages that brew already brings correct? thanks in advance!

**Reply by SMillerDev** [reply]
Yes, it refreshed the manpage files that brew came with.

**Reply by RogerioSoares96** [reply]
thank you so much @SMillerDev much appreciated for your clarification !

**Reply by unknowledgeable** [reply]
Oops, I've just done this by accident too, installed 31 Gemfile dependencies and tapped homebrew/bundle, homebrew/services and homebrew/test-bot.

Is there any simple way of reverting this process? 

Is it as easy as deleting the files in mentioned here:
```
Bundle complete! 31 Gemfile dependencies, 82 gems now installed.

Bundled gems are installed into `../../usr/local/Homebrew/Library/Homebrew/vendor/bundle`
```
and these directories:
```
Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle'...
Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-services'...
Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-test-bot'...
```

Or would that risk messing with the homebrew installation?

[Here's a gist of the full output](https://gist.github.com/unknowledgeable/aa0141e5b245ba7d966f63d34efda327)

Many thanks

**Reply by SMillerDev** [reply]
I'd leave the gems and just run:
```sh
brew untap homebrew/bundle
brew untap homebrew/services
brew untap homebrew/test-bot
```

**Reply by carlocab** [reply]
If you _must_ remove the gems, you can try (please read caveats at the end)

```
rm -rf ~/.gem # This may also delete any gems you installed by yourself
git -C $(brew --repo) clean -xdf # This will destroy any changes to untracked files in your HOMEBREW_REPOSITORY
brew update-reset # Read the output of `brew help update-reset` before running this
```

If you're not willing to do all three commands, then don't run any of them at all. And I especially wouldn't try the `git clean` command if I weren't also going to do `brew update-reset`.
